btree/btree.c

```diff
@@ -1797,7 +1788,10 @@ static int64_t node_get(struct node_op *op, struct td *tree,
 	struct nd              *node;
 	bool                    in_lrulist;
 
-	nxt_state =  segops->so_node_get(op, tree, addr, nxt);
+	nxt_state = nxt;
+	if (tree == NULL) {
+		nxt_state = mem_tree_get(op, addr, nxt);
+	}
```

Comment:
No need for nxt_state varaible ,  mem_tree_get is returning the same state which u are passing as an argument.

Reply:
ok